src/main/java/org/prgrms/vouchermanager/blacklist/Blacklist.java

```diff
@@ -0,0 +1,31 @@
+package org.prgrms.vouchermanager.blacklist;
+
+import java.util.UUID;
+
+public class Blacklist {
+    private final UUID customerId;
+    private final String email;
+
+    public Blacklist(UUID customerId, String email) {
+        if (email.isBlank()) throw new IllegalArgumentException("email은 공백이 될 수 없습니다.");
+        if (customerId == null) throw new IllegalArgumentException("blacklist의 UUID는 null이 될 수 없습니다.");
+        this.customerId = customerId;
+        this.email = email;
```

Comment:
```suggestion
        if (email.isBlank()) throw new IllegalArgumentException("email은 공백이 될 수 없습니다.");
        if (customerId == null) throw new IllegalArgumentException("blacklist의 UUID는 null이 될 수 없습니다.");
        
        this.customerId = customerId;
        this.email = email;
```

공백으로 문맥이 바뀜을 표현해주는것도 좋을거같아요